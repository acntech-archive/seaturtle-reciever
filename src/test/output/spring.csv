Percentage served,Time in ms
0,0.902
1,1.532
2,1.732
3,1.903
4,1.977
5,2.083
6,2.150
7,2.222
8,2.289
9,2.397
10,2.465
11,2.535
12,2.623
13,2.688
14,2.757
15,2.802
16,2.845
17,2.910
18,2.957
19,2.998
20,3.036
21,3.058
22,3.132
23,3.156
24,3.201
25,3.237
26,3.277
27,3.305
28,3.349
29,3.371
30,3.414
31,3.435
32,3.468
33,3.509
34,3.550
35,3.588
36,3.629
37,3.659
38,3.704
39,3.748
40,3.775
41,3.820
42,3.855
43,3.887
44,3.914
45,3.953
46,3.992
47,4.033
48,4.064
49,4.124
50,4.163
51,4.192
52,4.240
53,4.304
54,4.367
55,4.411
56,4.460
57,4.504
58,4.555
59,4.585
60,4.623
61,4.676
62,4.732
63,4.779
64,4.836
65,4.901
66,4.929
67,5.001
68,5.034
69,5.089
70,5.161
71,5.211
72,5.276
73,5.379
74,5.485
75,5.542
76,5.667
77,5.750
78,5.782
79,5.978
80,6.108
81,6.269
82,6.398
83,6.473
84,6.646
85,6.817
86,7.136
87,7.313
88,7.510
89,7.858
90,8.107
91,8.348
92,8.702
93,9.262
94,9.903
95,10.183
96,10.718
97,11.727
98,13.486
99,21.271
100,23.102
