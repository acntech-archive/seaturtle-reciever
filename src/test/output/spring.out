This is ApacheBench, Version 2.3 <$Revision: 1706008 $>
Copyright 1996 Adam Twiss, Zeus Technology Ltd, http://www.zeustech.net/
Licensed to The Apache Software Foundation, http://www.apache.org/

Benchmarking localhost (be patient)
Completed 100 requests
Completed 200 requests
Completed 300 requests
Completed 400 requests
Completed 500 requests
Completed 600 requests
Completed 700 requests
Completed 800 requests
Completed 900 requests
Completed 1000 requests
Finished 1000 requests


Server Software:        
Server Hostname:        localhost
Server Port:            8080

Document Path:          /seaturtle/api/heartbeat
Document Length:        2263 bytes

Concurrency Level:      10
Time taken for tests:   0.496 seconds
Complete requests:      1000
Failed requests:        0
Keep-Alive requests:    0
Total transferred:      2382000 bytes
HTML transferred:       2263000 bytes
Requests per second:    2016.51 [#/sec] (mean)
Time per request:       4.959 [ms] (mean)
Time per request:       0.496 [ms] (mean, across all concurrent requests)
Transfer rate:          4690.75 [Kbytes/sec] received

Connection Times (ms)
              min  mean[+/-sd] median   max
Connect:        0    1   1.0      0      12
Processing:     1    4   2.8      4      23
Waiting:        0    4   2.7      3      23
Total:          1    5   2.9      4      23

Percentage of the requests served within a certain time (ms)
  50%      4
  66%      5
  75%      6
  80%      6
  90%      8
  95%     10
  98%     13
  99%     21
 100%     23 (longest request)
